tox.ini

```diff
@@ -95,7 +95,6 @@ commands =
 
 [testenv:changelog]
 basepython = python3
-skip_install = True
```

Comment:
What's this change about? It seems unrelated to the PR's description.

Reply:
Twine needs to be installed for towncrier to work properly. This had been added and needed to be removed

Reply:
Unsatisfied, I looked at the history. I see it was added in 0191f0c9d9cae285df4c700dece7efc7c7de1551 without any rationale, so if it's needed, I guess maybe re-adding it makes sense.

> Twine needs to be installed for towncrier to work properly.

That sounds wrong to me. I use towncrier in other projects without problems. I see now you mean is that that towncrier, as [configured for this project](https://github.com/pypa/twine/blob/6fbf880ee60915cf1666348c4bdd78a10415f2ac/pyproject.toml#L74), needs the package to be installed.